lms/djangoapps/course_home_api/dates/tests/test_views.py

```diff
@@ -20,7 +20,7 @@ class DatesTabTestViews(BaseCourseHomeTests):
     """
     def setUp(self):
         super().setUp()
-        self.url = reverse('course-home-dates-tab', args=[self.course.id])
+        self.url = reverse('course-home:dates-tab', args=[self.course.id])
```

Comment:
What's happening with this change? I might not know enough about reverse.

Reply:
This was just me being ornery. Every current url in `course_home_urls` had to specify the `course-home-` prefix themselves (and one had a `course-experience-` prefix, which felt wrong).

So to avoid extra typing and other mistakes, I just used a namespace instead at the `lms/urls.py` level. And namespaces use the colon, so I had to change all references.